### PR TITLE
Update forms 1 to 1

### PR DESCRIPTION
**Pull request by paulfalgout**
- [x] Add the Clubhouse Story ID: [ch21965] [ch21980] [ch21981]

Also includes some fixes for picklist and the search picklist

### CODE REVIEW

File: src/js/behaviors/picklist-transport.js [hunk 1]
```diff
@@ -116,28 +117,34 @@ export default Behavior.extend({
...
+    const childViewHeight = $scrollItem.outerHeight();
+    const childViewTop = $scrollItem.position().top;
     const childViewBottom = childViewTop + childViewHeight - picklistHeight;
     let offset = 0;
 
     if (offsetDir === 'middle') {
       offset = (picklistHeight / 2) - (childViewHeight / 2);
     }
-
+
```
Comment (luketlancaster) [inline src/js/behaviors/picklist-transport.js]: extra spaces here

Comment (paulfalgout) [reply src/js/behaviors/picklist-transport.js]: my .editorconfig plugin stopped working with the my latest update :-/

**Comment by paulfalgout** [comment]
@luketlancaster @LisaManresa since I'm out if someone wants to pick up on this with looking into these tests, that'd be good.  I think in a few cases I missed updating test data to match 1-to-1 forms.

**Review by luketlancaster — APPROVED** [review]
lookin good to me. I'm guessing this is good to go since all the checks have passed

**Comment by paulfalgout** [comment]
Screen Shot 2020-02-12 at 1 10 35 AM
Looking for Mark's opinion on the "clear" before this goes in.

**Pull request merged**